tasks/compare-compiled-output.js

```diff
@@ -44,10 +41,12 @@ env.addFilter("print", (data) => JSON.stringify(data, null, 2));
 const log = {
 	error: (err) => process.stderr.write(`${err}\n\n`),
 	write: (msg) => process.stdout.write(msg),
-	writeTable: (data, padding = 30) => {
```

Comment:
Updated this to find the package rather than assume where it lives in the repo and to better pad the console output for longer dist paths like the tokens use.